### PR TITLE
Feature/coding standards

### PR DESCRIPTION
**Pull request by ethan-schaffer**
Updates to genesis-simple-edits to conform to Wordpress PHP standards and implement CircleCI

### CODE REVIEW

File: class-genesis-simple-edits.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+ * @since 1.0
+ */
+class Genesis_Simple_Edits {
+
+	/**
+	 * Plugin version
+	 *
+	 * @var $plugin_version Plugin Version
+	 */
+	public $plugin_version = '2.2.1';
```
Comment (marksabbath) [inline class-genesis-simple-edits.php]: We could define a constant in the main plugin file, like `GENESIS_SIMPLE_EDITS_VERSION` and use here as well.

File: includes/class-genesis-simple-edits.php [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	}
+
+	/**
+	 * Include the class file, instantiate the classes, create objects.
+	 *
+	 * @since 2.2.0
+	 */
+	public function instantiate() {
+
+		require_once $this->plugin_dir_path . 'class-genesis-simple-edits-core.php';
```
Comment (marksabbath) [inline includes/class-genesis-simple-edits.php]: This is one line that should have the path fixed (adding the `include/`)

File: includes/class-genesis-simple-edits.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		$this->core = new Genesis_Simple_Edits_Core();
+		$this->core->init();
+
+		if ( is_admin() ) {
+			require_once $this->plugin_dir_path . 'class-genesis-simple-edits-admin.php';
```
Comment (marksabbath) [inline includes/class-genesis-simple-edits.php]: This is one line that should have the path fixed (adding the `include/`)

File: includes/views/admin.php [hunk 4]
```diff
@@ -1,88 +1,106 @@
...
 		</td>
 	</tr>
 
 	<tr class="footer-shortcodes" style="display: none;">
-		<th scope="row"><p><span class="description"><?php _e( 'Shortcode Reference', 'genesis-simple-edits' ); ?></span></p></th>
+		<th scope="row"><p><span class="description"><?php esc_html_e( 'Shortcode Reference', 'genesis-simple-edits' ); ?></span></p></th>
 		<td>
-			<p><span class="description"><?php printf( __( 'NOTE: For a more comprehensive shortcode usage guide, see the <a href="%s" target="_blank">footer shortcode reference</a>.', 'genesis-simple-edits' ), 'http://my.studiopress.com/documentation/customization/shortcodes-reference/footer-shortcode-reference/' ); ?></span></p>
+			<p><span class="description"><?php // Translators: The string is a URL to a usage guide. ?><?php printf( esc_attr( __( 'NOTE: For a more comprehensive shortcode usage guide, see the <a href="%s" target="_blank">footer shortcode reference</a>.', 'genesis-simple-edits' ) ), 'http://my.studiopress.com/documentation/customization/shortcodes-reference/footer-shortcode-reference/' ); ?></span></p>
```
Comment (dreamwhisper) [inline includes/views/admin.php]: Here, we're using `esc_attr` but on line 40+, there's different formatting and `wp_kses` is used.

**Pull request merged**